### ISSUE TITLE
Using addDDBIEffectToDocument checks for an undefined useChrisPremades

### ISSUE THREAD

**Issue by Krigsmaskine** [MrPrimate/ddb-importer#322]
**Describe the bug**
Hi, I'm not a user of DnDBeyond and I tried the sample macro you had for using one of your item setups and I get this error in the console when trying to run it;
 
```
helpers.js:31 Uncaught (in promise) TypeError: undefined. Cannot read properties of undefined (reading 'useChrisPremades')
[Detected 3 packages: ddb-importer, advanced-macros, lib-wrapper]
    at Object.addDDBIEffectToDocument (helpers.js:31:59)
    at eval (eval at executeScript (lib.js:175:13), <anonymous>:5:53)
    at Macro.eval (eval at executeScript (lib.js:175:13), <anonymous>:6:3)
    at Macro.executeScript (lib.js:179:13)
    at 🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:591:14)
    at 🎁Macro.prototype._executeScript#0 (libWrapper-wrapper.js:189:20)
    at Macro.renderMacro [as renderContent] (lib.js:293:17)
    at _executeMacroInternal (lib.js:137:17)
    at Macro.executeMacro (lib.js:77:15)
    at 🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:591:14)
    at 🎁Macro.prototype.execute#0 (libWrapper-wrapper.js:189:20)
    at Hotbar._onClickMacro (foundry.js:68365:20)
    at HTMLLIElement.dispatch (jquery.min.js:2:43064)
    at y.handle (jquery.min.js:2:41048)
```

**To Reproduce**
1. Use the example macro for an item (in my case it was Vicious Mockery);
```
const TestItem = game.items.getName("Vicious Mockery");
await game.modules.get("ddb-importer")?.api.effects.addDDBIEffectToDocument(TestItem);
```
2. Macro does not run, see console output similar to error above.

**Expected behavior**

**Screenshots**

**Environment:**
 - Local Foundry client.
 - Foundry 10.291
 - 5e 2.1.5
 - DDB-Importer version 3.4.17
 - Does this behaviour occur with other modules disabled: Yes

**Comment by MrPrimate** [comment]
These are less sample macros, and macros that are used by DDB Importer, where it uses some functions provided by the importer. If you want to use the macro, you will need to extract those functions into your macro and replace the code, or have DDB Importer running.

**Comment by Krigsmaskine** [comment]
> 

The module (and Chris') is on, but the modules macro is checking a variable that isn't defined

**Comment by MrPrimate** [comment]
> The _module_ is on, I just wanted to stress I wasn't able to use the importer's GUI as I don't use the website DnDBeyond, but the modules' macro is checking a variable that isn't defined. It's the macro from the FAQ for specifically this use case: https://github.com/MrPrimate/ddb-importer/blob/main/FAQ.md#i-want-to-use-your-macroseffectsetc-but-dont-use-ddb

Apologies, I misread this (too early and not enough coffee!) I'll issue a fix later today.

**Comment by MrPrimate** [comment]
Fixed in v3.4.18
